calico_node/startup/startup.go

```diff
@@ -341,6 +341,22 @@ func validateIP(ipn *net.IPNet) {
 	warning("Unable to confirm IPv%d address %s is assigned to this host", ip.Version(), ip)
 }
 
+// evaluateENVBool evaluates a passed environment variable
+// Returns an boolean indicating if the parameter was defined
+// and an boolean if the argument was true or false
+// If it was false an additional log entry will be written
+
+func evaluateENVBool(envvar string, logentry string) (isDefined bool, outgoingNATEnabled bool) {
+	envValue, isEmpty := os.LookupEnv(envvar)
+	switch strings.ToLower(envValue) {
+	case "false", "0", "no", "n", "f":
+		log.Info(logentry)
```

Comment:
I think the logging would be cleaner if instead of passing in the log entry, we just logged something like this:

```
log.Info("%s (%s) is disabled through environment variable", envVar, envValue)
```

Then we won't need to pass in the logs throughout, and all the logs will be guaranteed to have the same format.